scipy/linalg/special_matrices.py

```diff
@@ -1190,3 +1190,121 @@ def fiedler_companion(a):
     c[[0, 1], 0] = [-a[1], 1]
 
     return c
+
+
+def convolution_matrix(a, n, mode='full'):
+    """
+    Construct a convolution matrix.
+
+    Constructs the dense matrix representing one-dimensional
+    convolution [1]_.
+
+    ``A = convolution_matrix(a, n[, mode])`` creates a matrix ``A`` such
+    that ``A @ v`` (or ``matmul(A, v)``) is equivalent to using
+    ``convolve(a, v[, mode])``. In the default 'full' mode, the entries of
+    ``A`` are given by ``A[i, j] == (a[i-j] if (0 <= (i-j) < m) else 0)``.
+
+    Parameters
+    ----------
+    a : (m,) array_like
+        The 1-D array to convolve.
+    n : int
+        The number of columns in the resulting matrix.
+        This is analogous to the length of `v` in ``numpy.convolve(a, v)``.
```

Comment:
double backticks around `v` and maybe you can put single quotes around numpy.convolve for auto-linking (without the arguments obviously)